package.json

```diff
@@ -10,7 +10,7 @@
   "repository": "pillarjs/hbs",
   "main": "lib/hbs.js",
   "dependencies": {
-    "handlebars": "4.3.5",
+    "handlebars": "4.5.3",
```

Comment:
@dougwilson Would it make sense to pin it to the major version so people can get updates independently? Alternatively, change it to a peer dependency?

Reply:
Handlebars has not been following semver and there have been several non major releases where parts of this module had to be changed in order to work with the new handlebars.

Reply:
Ah, that's unfortunate - thanks!

Reply:
Yea. It seems to be common among libs that started out as client side and then just publish to npm. jQuery, lodash, etc. all seem similar in that regard as well.